src/vcd/meshing/mesh.py

```diff
@@ -90,4 +91,5 @@ def write(self, filename: str, clusters: List[trimesh.Trimesh]) -> None:
                     cluster.triangles,
                     partTypes=[TRIANGLE_STRIP] * len(cluster.triangles),
                 )  # one type for each part
-                w.record(cluster.volume, cluster.area, cluster.cluster_id)
+                is_ground = cluster.cluster_id == 2
```

Comment:
I think is_ground is true whether or not the points have Classification==2, not their cluster_id is 2. That's just the 2nd cluster that was made.

Reply:
🤦🏻‍♂️ why yes.... the classification, not the cluster_id .... ugh yeah I'll fix that.

Reply:
Fixed in d03b0903901a923bd2fa28fdb059e7b77a352133